llvm/lib/CheerpUtils/IdenticalCodeFolding.cpp

```diff
@@ -844,6 +849,29 @@ bool IdenticalCodeFolding::isStaticIndirectFunction(const llvm::Value* A)
 	return ce && ce->isCast() && isa<Function>(ce->getOperand(0));
 }
 
+bool IdenticalCodeFolding::isFunctionExternal(const llvm::Function* F)
+{
+	if (externalFunctionMapping.count(F))
```

Comment:
This lookup is double. Prefer `auto it = externalFunctionMapping.find(F); if(it != externalFunctionMapping.end()) return it->second;`

Reply:
Thanks, updated